draft-gpew-priv-ppm.md

```diff
@@ -445,9 +474,7 @@ number generator. Each task has the following parameters associated with it:
 
 * `aggregator_endpoints`: A list of URLs relative to which an aggregator's API
   endpoints can be found. Each endpoint's list MUST be in the same order. The
-  leader's endpoint MUST be the first in the list. The order of the
-  `encrypted_input_shares` in a `Report` (see {{uploading-reports}}) MUST be the
-  same as the order in which aggregators appear in this list.
+  leader's endpoint MUST be the first in the list.
```

Comment:
Is this requirement necessary anymore? It seems like the client doesn't need to care which aggregator is the leader if both are treated as separate aggregators to store shares.

Reply:
Oh, yeah, that was totally not clear to me! How is this expressed in terms of VDAF?

Reply:
It's not explicit in the syntax, but for schemes that needs to differentiate aggregator roles, they can do so via the verification parameter. For example, see https://cjpatton.github.io/vdaf/draft-patton-cfrg-vdaf.html#name-setup-2